pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -65,12 +65,8 @@
               = I18n.t(entry[:text_h1])
             %p
               = I18n.t(entry[:text_desc])
-            - if entry[:studio_button_link]
-              %a.link-button{href: CDO.studio_url(entry[:studio_button_link])}
-                = I18n.t(entry[:button_text])
-            - else
-              %a.link-button{href: entry[:button_link]}
-                = I18n.t(entry[:button_text])
+            %a.link-button{href: entry[:studio_button_link] ? CDO.studio_url(entry[:studio_button_link]) : entry[:button_link], class: [(entry[:external_link] ? "has-external-link" : nil), (entry[:button_color_white] ? "white" : nil)].compact.join(' '), target: (entry[:external_link] ? "_blank" : nil), rel: (entry[:external_link] ? "noopener noreferrer" : nil)}
```

Comment:
I know this is messy, but it adds the ability to add an external link and a white button in the `homepage.json` file.